ui/shared/status/StatusIcon.qml

```diff
@@ -0,0 +1,29 @@
+import QtQuick 2.13
+import QtGraphicalEffects 1.13
+
+Image {
+    property string icon: ""
+    property color color
+
+    id: root
+    width: 24
+    height: 24
```

Comment:
The same think: maybe implicit sizes? :)

Reply:
Turns out `implicitWidth` and `implicitHeight` are read-only properties on `Image`, so we can't set them

Reply:
As I'm working on #2443 I realised I need to set `sourceSize.width` and `sourceSize.height` here. Gonna mark as not merging until I've updated this PR.

Reply:
Updated it to set `sourceSize.width|height`